tests/units/csl/test_init_sanitizer.cpp

```diff
@@ -3,7 +3,9 @@
 #include <csl/initSanitizer.h>
 #include <gtest/gtest.h>
 #include <cmath>
-#include <string.h>
+// #include <string.h>
+
+#define strcmp(X,Y) (Y==X)

```

Comment:
These lines can be removed, right ?

Reply:
Here I re-defined strcmp for taking two `std::string` and returning 0 for true, 1 for false.

In the same file I also edited the comparison condition for returning `false` and not an exception for the comparison of an unassigned variable. I commented the old code, if that's ok I can delete it and push the commit.